config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

```diff
@@ -0,0 +1,39 @@
+presubmits:
+  kubernetes-sigs/kueue:
+  - name: pull-kueue-build
+    always_run: true
+    skip_branches:
+    - ahg-g-patch-1
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-kueue
```

Comment:
sig-scheduling

Reply:
mega face palm, sorry I took the template from the NFD project, fixing.....